clase6/ej1.js

```diff
@@ -0,0 +1,80 @@
+const $botonMiembros = document.querySelector("#botonMiembros");
+const $formulario = document.querySelector("#formulario");
+const $cantMiembros = document.querySelector("#cantMiembros");
+const $edadMayor = document.querySelector("#edadMayor");
+const $edadMenor = document.querySelector("#edadMenor");
+const $edadPromedio = document.querySelector("#edadPromedio");
+const $botonCalcular = document.querySelector("#botonCalcular");
+const $salarioMayor = document.querySelector("#salarioMayor");
+const $salarioMenor = document.querySelector("#salarioMenor");
+const $salarioPromedio = document.querySelector("#salarioPromedio");
+
+$botonMiembros.onclick = function () {
+  const cantMiembros = Number($cantMiembros.value);
+
+  for (let i = 0; i < cantMiembros; i++) {
+    const $nuevaLabelEdad = document.createElement("label");
+    $nuevaLabelEdad.textContent = `Ingrese edad del ${i + 1}° integrante: `;
+
+    const $nuevoInputEdad = document.createElement("input");
+    $nuevoInputEdad.type = "number";
+    $nuevoInputEdad.id = "edades";
+
+    const $nuevaLabelSalario = document.createElement("label");
+    $nuevaLabelSalario.textContent = `Ingrese el salario mensual del ${
+      i + 1
+    }° integrante`;
+
+    const $nuevoInputSalario = document.createElement("input");
+    $nuevoInputSalario.type = "number";
+    $nuevoInputSalario.id = "salarios";
+
+    const $br = document.createElement("br");
```

Comment:
Podrías ahorrarte el `br` con css, poniendole a la clase padre de los elementos un estilo, te dejo unos links de flexbox para que chusmees:
https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox
https://flexboxfroggy.com/#es
https://css-tricks.com/snippets/css/a-guide-to-flexbox/